css/style.css

```diff
@@ -220,14 +225,18 @@ main {
   border-radius: 2px;
 }
 
-.toggle-inner {
+input[type="checkbox"] {
+  display: none;
```

Comment:
Okay I am understanding you now. Thank you for the reply! I have made the changes.